components/ChoreWeight.tsx

```diff
@@ -104,7 +116,7 @@ const s = StyleSheet.create({
     marginHorizontal: 5,
   },
   circleText: {
-    color: 'black',
+    // color: 'black',
```

Comment:
Honestly, hardcoding this to black is fine, that's what looks best in both dark and light mode tbh (for this particular thing)
```suggestion
    color: 'black',
```